### PR TITLE
Automatic RHMP integration

### PR DESCRIPTION
**Pull request by Niflhem**

### CODE REVIEW

**Comment by kacperkrzyzak** [comment]
I think you can also add logging of RHMP feature status here: controllers/ibmlicenseservicereporter_controller.go:478

**Review by kacperkrzyzak — APPROVED** [review]

File: controllers/resources/helper.go [hunk 1]
```diff
@@ -331,6 +333,13 @@ func UpdateCacheClusterExtensions(client c.Reader) error {
 		c.InNamespace(namespace),
 	}
 
+	MeterDefinitionCRD := &rhmp.MeterDefinition{}
+	if err := client.List(context.TODO(), MeterDefinitionCRD, listOpts...); err == nil {
+		RHMPEnabled = true
```
Comment (pgodowski) [inline controllers/resources/helper.go:338]: Pls add status

**Review by pgodowski — CHANGES_REQUESTED** [review]

File: controllers/resources/helper.go (hunk 1 above)
Comment (Niflhem) [reply controllers/resources/helper.go:338]: Fixed here https://github.com/IBM/ibm-licensing-operator/pull/533/commits/cd680066c47a857698c0574d044642aa2319b1cf

**Review by adamdyszy — APPROVED** [review]

**Pull request merged**